src/main/java/seedu/address/ui/PetDisplay.java

```diff
@@ -0,0 +1,52 @@
+package seedu.address.ui;
+
+import java.io.File;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Comparator;
+import javafx.fxml.FXML;
+import javafx.scene.layout.HBox;
+import javafx.scene.layout.Region;
+import javafx.scene.image.Image;
+import javafx.scene.image.ImageView;
+
+
+/** An UI component that displays image of a Pet {@code Pet}. */
+public class PetDisplay extends UiPart<Region> {
```

Comment:
Did not know you can extend it from UiPart like that, good learning point for us, thanks Permas